app/src/main/res/navigation/nav_graph_onboarding.xml

```diff
@@ -80,12 +89,39 @@
         <action
             android:id="@+id/action_mapFragment_to_navigation_intro"
             app:destination="@id/navigation_intro" />
-        <argument
-            android:name="receivedResponse"
-            app:argType="com.internship.move.feature.authentication.UserResponse" />
     </fragment>
     <fragment
```

Comment:
Format this file to have empty lines between xml tags, similar to how you do it in UI xmls